tests/apps/configurationapp/app.go

```diff
@@ -0,0 +1,274 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/dapr/components-contrib/configuration"
+	"github.com/dapr/components-contrib/tests/utils/configupdater"
+	cu_redis "github.com/dapr/components-contrib/tests/utils/configupdater/redis"
+	"github.com/dapr/dapr/tests/apps/utils"
+	"github.com/dapr/kit/logger"
+	"github.com/gorilla/mux"
```

Comment:
Since this is a new test, could you switch to Chi right away? gorilla/mux is unmaintained and we'll have to migrate to Chi in the not-too-distant future